src/Relation/Binary/Bundles.agda

```diff
@@ -152,7 +155,7 @@ record DecPoset c ℓ₁ ℓ₂ : Set (suc (c ⊔ ℓ₁ ⊔ ℓ₂)) where
     }
 
   open Poset poset public
-    using (preorder)
+    using (preorder; _≰_)
```

Comment:
NB we can do nothing about the fact that `preorder` will have (redundant!) fields names not congruent with those renamed in `Poset`... is that a problem in practice? In theory?

Reply:
Here's how I understand it:
In the substructure `preorder` of `Poset`  (each inside `DecPoset`), *even though the base relation* has been passed in with one symbol `_≤_`, this export of the substructure will expose its field name with the `Preorder` symbol `_≲_`, even though references to that symbol, *qua* field name, will nevertheless be definitionally indistinguishable from its associated binding `_≤_`... suggesting at the very least (?) that the reference `_≲_` should be deprecated? Or is everything *really*/'really' OK after all (not least because `Preorder`-specific operations may even expect a field called `_≲_``...)

Reply:
I mean we're not opening `preorder` here so we don't get the `_≲_` relation added to the scope. If the user really wants to open the `preorder` bundle themselves, then yes, they will get `_≲_` exposed, but I don't see why anyone would do so, as opening `DecPoset` will bring everything into scope with the correct names.

So not a problem in practice I think :+1: